src/user/models/user-data/__tests__/mock/user-data.entity.mock.ts

```diff
@@ -20,6 +20,7 @@ export function createCustomUserData(customValues: Partial<UserData>): UserData
   const entity = new UserData();
 
   entity.mail = keys.includes('mail') ? mail : 'test@test.com';
+  entity.country = createDefaultCountry();
 
   return entity;
```

Comment:
Use the same logic as for country mock here (add a default UserData and use the Object.assign())